akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -186,6 +186,12 @@ object Source {
       override def toString: String = "() => Iterator"
     })
 
+  def fromJavaStream[T, S <: java.util.stream.BaseStream[T, S]](stream: () ⇒ java.util.stream.BaseStream[T, S]): Source[T, Unit] = {
```

Comment:
The reason why I put here, because it is a natural place on Source for the `javadsl` and I wanted to be symmetric.